script.js

```diff
@@ -113,4 +115,27 @@ contForm.addEventListener('submit', (event) => {
   } else {
     errorMessage.style.display = 'block';
   }
-});
\ No newline at end of file
+});
+
+function setText() {
+  const contactFormInfo = JSON.parse(localStorage.getItem('info'));
+  firstName.value = contactFormInfo.firstName;
+  emailForm.value = contactFormInfo.email;
+  textArea.value = contactFormInfo.textArea;
+}
+
+function populateStorage() {
+  localStorage.setItem('info', JSON.stringify({ firstName: firstName.value, email: emailForm.value, textArea: textArea.value }));
```

Comment:
Hello @chaw-bot, 
As you can see on this line, it is already stored as a single object, and the line that you are commenting on is for the output after the object is converted for another function. Can you please clarify your feedback and add some advice or recommendations for improving this feature. Thanks for your attention.